[923:MainThread](2021-10-22 01:01:59,951) INFO - qlib.Initialization - [config.py:386] - default_conf: client.
[923:MainThread](2021-10-22 01:01:59,953) WARNING - qlib.Initialization - [config.py:411] - redis connection failed(host=127.0.0.1 port=6379), DiskExpressionCache and DiskDatasetCache will not be used!
[923:MainThread](2021-10-22 01:01:59,955) INFO - qlib.Initialization - [__init__.py:56] - qlib successfully initialized based on client settings.
[923:MainThread](2021-10-22 01:01:59,955) INFO - qlib.Initialization - [__init__.py:58] - data_path={'__DEFAULT_FREQ': PosixPath('/home/us_data')}
[923:MainThread](2021-10-22 01:01:59,956) INFO - qlib.workflow - [expm.py:270] - No tracking URI is provided. Use the default tracking URI.
[923:MainThread](2021-10-22 01:01:59,957) INFO - qlib.workflow - [expm.py:306] - <mlflow.tracking.client.MlflowClient object at 0x7fd56b0b66d0>
[923:MainThread](2021-10-22 01:01:59,988) INFO - qlib.workflow - [exp.py:249] - Experiment 1 starts running ...
[923:MainThread](2021-10-22 01:02:00,135) INFO - qlib.workflow - [recorder.py:284] - Recorder 971be2dd43884bbfa1847ed24ecc20e8 starts running under Experiment 1 ...
[923:MainThread](2021-10-22 01:02:00,190) INFO - qlib.workflow - [expm.py:270] - No tracking URI is provided. Use the default tracking URI.
[923:MainThread](2021-10-22 01:02:00,191) INFO - qlib.workflow - [expm.py:306] - <mlflow.tracking.client.MlflowClient object at 0x7fd56b0b67f0>
[923:MainThread](2021-10-22 01:02:00,192) INFO - qlib.workflow - [exp.py:249] - Experiment 1 starts running ...
[923:MainThread](2021-10-22 01:02:00,206) INFO - qlib.workflow - [recorder.py:284] - Recorder 971be2dd43884bbfa1847ed24ecc20e8 starts running under Experiment 1 ...
/usr/local/lib/python3.8/dist-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /pytorch/c10/cuda/CUDAFunctions.cpp:100.)
  return torch._C._cuda_getDeviceCount() > 0
[923:MainThread](2021-10-22 01:02:02,114) WARNING - qlib.TRA - [pytorch_tra.py:100] - `eval_train` will be ignored when using TRA.router
[923:MainThread](2021-10-22 01:02:02,121) INFO - qlib.TRA - [pytorch_tra.py:141] - init TRAModel...
[923:MainThread](2021-10-22 01:02:02,139) INFO - qlib.TRA - [pytorch_tra.py:171] - # model params: 218368
[923:MainThread](2021-10-22 01:02:02,140) INFO - qlib.TRA - [pytorch_tra.py:172] - # tra params: 6374
Please install necessary libs for CatBoostModel.
Please install necessary libs for XGBModel, such as xgboost.
RNN(
  (rnn): LSTM(6, 128, num_layers=2, batch_first=True)
  (W): Linear(in_features=128, out_features=128, bias=True)
  (u): Linear(in_features=128, out_features=1, bias=False)
)
TRA(
  (predictors): Linear(in_features=256, out_features=3, bias=True)
  (router): LSTM(3, 32, batch_first=True)
  (fc): Linear(in_features=288, out_features=3, bias=True)
)
[923:MainThread](2021-10-22 01:05:09,716) INFO - qlib.timer - [log.py:113] - Time cost: 187.574s | Loading data Done
Killed
/usr/local/lib/python3.8/dist-packages/joblib/externals/loky/backend/resource_tracker.py:318: UserWarning: resource_tracker: There appear to be 2 leaked folder objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
/usr/local/lib/python3.8/dist-packages/joblib/externals/loky/backend/resource_tracker.py:333: UserWarning: resource_tracker: /tmp/joblib_memmapping_folder_923_ff6287762045463d9339b66e72159042_26cfa8ff7449472b90eee5bb6cf7ad5d: FileNotFoundError(2, 'No such file or directory')
  warnings.warn('resource_tracker: %s: %r' % (name, e))
/usr/local/lib/python3.8/dist-packages/joblib/externals/loky/backend/resource_tracker.py:333: UserWarning: resource_tracker: /tmp/joblib_memmapping_folder_923_9ae37c9cf3d840d586be5bd1ae38e84e_32d33d96dd654d46a7a3b62d9be0c058: FileNotFoundError(2, 'No such file or directory')
  warnings.warn('resource_tracker: %s: %r' % (name, e))
[1094:MainThread](2021-10-22 09:45:28,106) INFO - qlib.Initialization - [config.py:386] - default_conf: client.
[1094:MainThread](2021-10-22 09:45:28,110) WARNING - qlib.Initialization - [config.py:411] - redis connection failed(host=127.0.0.1 port=6379), DiskExpressionCache and DiskDatasetCache will not be used!
[1094:MainThread](2021-10-22 09:45:28,112) INFO - qlib.Initialization - [__init__.py:56] - qlib successfully initialized based on client settings.
[1094:MainThread](2021-10-22 09:45:28,112) INFO - qlib.Initialization - [__init__.py:58] - data_path={'__DEFAULT_FREQ': PosixPath('/home/us_data')}
[1094:MainThread](2021-10-22 09:45:28,113) INFO - qlib.workflow - [expm.py:270] - No tracking URI is provided. Use the default tracking URI.
[1094:MainThread](2021-10-22 09:45:28,114) INFO - qlib.workflow - [expm.py:306] - <mlflow.tracking.client.MlflowClient object at 0x7f80e6ebb6a0>
[1094:MainThread](2021-10-22 09:45:28,146) INFO - qlib.workflow - [exp.py:249] - Experiment 1 starts running ...
[1094:MainThread](2021-10-22 09:45:28,299) INFO - qlib.workflow - [recorder.py:284] - Recorder 155d2ae7d47c4252b8bde18a7aafe669 starts running under Experiment 1 ...
[1094:MainThread](2021-10-22 09:45:28,354) INFO - qlib.workflow - [expm.py:270] - No tracking URI is provided. Use the default tracking URI.
[1094:MainThread](2021-10-22 09:45:28,354) INFO - qlib.workflow - [expm.py:306] - <mlflow.tracking.client.MlflowClient object at 0x7f80e6ebb7c0>
[1094:MainThread](2021-10-22 09:45:28,355) INFO - qlib.workflow - [exp.py:249] - Experiment 1 starts running ...
[1094:MainThread](2021-10-22 09:45:28,370) INFO - qlib.workflow - [recorder.py:284] - Recorder 155d2ae7d47c4252b8bde18a7aafe669 starts running under Experiment 1 ...
/usr/local/lib/python3.8/dist-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /pytorch/c10/cuda/CUDAFunctions.cpp:100.)
  return torch._C._cuda_getDeviceCount() > 0
[1094:MainThread](2021-10-22 09:45:30,279) WARNING - qlib.TRA - [pytorch_tra.py:100] - `eval_train` will be ignored when using TRA.router
[1094:MainThread](2021-10-22 09:45:30,284) WARNING - qlib.TRA - [pytorch_tra.py:132] - logdir output/us360_nassp_epoch500 is not empty
[1094:MainThread](2021-10-22 09:45:30,285) INFO - qlib.TRA - [pytorch_tra.py:141] - init TRAModel...
[1094:MainThread](2021-10-22 09:45:30,302) INFO - qlib.TRA - [pytorch_tra.py:171] - # model params: 55936
[1094:MainThread](2021-10-22 09:45:30,302) INFO - qlib.TRA - [pytorch_tra.py:172] - # tra params: 5606
Please install necessary libs for CatBoostModel.
Please install necessary libs for XGBModel, such as xgboost.
RNN(
  (rnn): LSTM(6, 64, num_layers=2, batch_first=True)
  (W): Linear(in_features=64, out_features=64, bias=True)
  (u): Linear(in_features=64, out_features=1, bias=False)
)
TRA(
  (predictors): Linear(in_features=128, out_features=3, bias=True)
  (router): LSTM(3, 32, batch_first=True)
  (fc): Linear(in_features=160, out_features=3, bias=True)
)
[1094:MainThread](2021-10-22 09:48:36,796) INFO - qlib.timer - [log.py:113] - Time cost: 186.491s | Loading data Done
Killed
/usr/local/lib/python3.8/dist-packages/joblib/externals/loky/backend/resource_tracker.py:318: UserWarning: resource_tracker: There appear to be 2 leaked folder objects to clean up at shutdown
  warnings.warn('resource_tracker: There appear to be %d '
/usr/local/lib/python3.8/dist-packages/joblib/externals/loky/backend/resource_tracker.py:333: UserWarning: resource_tracker: /tmp/joblib_memmapping_folder_1094_e6c88343de4846b1ba848aef7fe3abe6_72d2b7cfd7ff42038def55437d9a4432: FileNotFoundError(2, 'No such file or directory')
  warnings.warn('resource_tracker: %s: %r' % (name, e))
/usr/local/lib/python3.8/dist-packages/joblib/externals/loky/backend/resource_tracker.py:333: UserWarning: resource_tracker: /tmp/joblib_memmapping_folder_1094_349dcebe3d80475ea5cba5768b7a3f78_5602a2c70bd34f95a5db847e00cd5a29: FileNotFoundError(2, 'No such file or directory')
  warnings.warn('resource_tracker: %s: %r' % (name, e))
